src/sphinx_book_theme/header_buttons/launch.py

```diff
@@ -161,6 +162,19 @@ def add_launch_buttons(
             }
         )
 
+    if basthon_url:
+        gh = "https://raw.githubusercontent.com"
+        url = f"{basthon_url}/?from={gh}/{org}/{repo}/{branch}/{path_rel_repo}"
+        launch_buttons_list.append(
+            {
+                "type": "link",
+                "text": "Basthon",
+                "tooltip": "Launch on Basthon",
+                "icon": "_static/images/logo_basthon.svg",
```

Comment:
Is there a logo that is square and meant for an icon? I think this will be difficult to read as a small square format. E.g. look at the other icons:

![image](https://user-images.githubusercontent.com/1839645/185611388-e794e6da-0dd2-4387-b0bb-bb2da60ff93e.png)

Reply:
[An example of how it renders](https://cpge-itc.github.io/itc2/1_sql/pokemon.html)
I am afraid I don't have a better image

Reply:
Or maybe this one: https://basthon.fr/theme/assets/img/basthon-notebook.svg ?